save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Email.kt

```diff
@@ -0,0 +1,40 @@
+/**
+ * rendering for Email and Login management card
+ */
+
+package com.saveourtool.save.frontend.components.views.usersettings.right
+
+import com.saveourtool.save.frontend.components.inputform.InputTypes
+import com.saveourtool.save.frontend.components.views.usersettings.SettingsProps
+import com.saveourtool.save.frontend.components.views.usersettings.inputForm
+import com.saveourtool.save.frontend.components.views.usersettings.saveUser
+import com.saveourtool.save.frontend.utils.*
+import react.FC
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.hr
+import react.useState
+import web.cssom.ClassName
+
+val email: FC<SettingsProps> = FC { props ->
```

Comment:
```suggestion
val email: FC<SettingsProps> = FC { props ->
```

The name is too short for global property